src/FormLabel.js

```diff
@@ -56,7 +56,8 @@ const FormLabel = React.forwardRef(
       className,
       bsPrefix,
       srOnly && 'sr-only',
-      column && 'col-form-label',
+      typeof column === 'boolean' && 'col-form-label',
```

Comment:
this would set `col-form-label` even if we had `column={false}`. see e.g. how we calculate and use `expandClass` in https://github.com/react-bootstrap/react-bootstrap/blob/1bb3f41d2a9650b06402c4d3dcce50b9c19a860f/src/Navbar.js#L164-L165